app/components/advertisement-tool.js

```diff
@@ -31,100 +31,90 @@ export default Component.extend({
       full: true,
     },
     {
-      title: 'StudentenWegwijzer',
-      link: 'https://www.studentenwegwijzer.nl/enschede/studentenverenigingen/alpha/',
-      image: 'advertisements/studenten-wegwijzer.png',
-      full: true,
+      title: 'BeBo Parket',
+      link: 'https://www.beboparket.nl/',
+      image: 'advertisements/BeBo.png',
     },
     {
-      title: 'InShared',
-      link: 'https://www.inshared.nl/',
-      image: 'advertisements/inshared.png',
+      title: 'Boeketcadeau',
+      link: 'https://www.boeketcadeau.nl/',
+      image: 'advertisements/Boeketcadeau.png',
     },
     {
-      title: 'Led24',
-      link: 'https://www.led24.nl/led-verlichting/led-tl-buis/',
-      image: 'advertisements/led24.png',
+      title: 'Baktotaal',
+      link: 'https://www.baktotaal.nl/',
+      image: 'advertisements/Bouwhuis.png',
     },
     {
-      title: 'Noppies',
-      link: 'https://www.noppies.com/',
-      image: 'advertisements/noppies.png',
+      title: 'Comfort Producten',
+      link: 'https://www.comfort-producten.nl/nl/',
+      image: 'advertisements/Comfort_products.png',
```

Comment:
Image name is inconsistent with the rest. Most images are camelCase (write words together but use a capital letter at the start of each word), this one is snake_case (using underscores to separate words). Since most are camelCase, please rename convert all image names to camelCase (should be about 5 images or so).

Reply:
I made sure that all the image filenames are in the same convention I actually converted all of them to camelCase.